### PR TITLE
Suppoort all CAPI CRD API versions

### PR DESCRIPTION
**Pull request by AverageMarcus**
Signed-off-by: Marcus Noble <github@marcusnoble.co.uk>

### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by whites11 — APPROVED** [review]
wow, LGTM

**Review by njuettner — APPROVED** [review]
lgtm 🙂

**Review by tfussell — CHANGES_REQUESTED** [review]
Please ensure that these policies don't affect Kyverno on clusters that don't have `v1alpha4` or `v1beta1` defined. Also I recommend restarting Kyverno after updating the policies and checking the logs to see if there are any warnings or non-apparent errors.

**Comment by AverageMarcus** [comment]
> Please ensure that these policies don't affect Kyverno on clusters that don't have `v1alpha4` or `v1beta1` defined. Also I recommend restarting Kyverno after updating the policies and checking the logs to see if there are any warnings or non-apparent errors.

There's some errors when applying the policy manifest, but the errors are just more of some we already have for other policies. (They're more warnings in my opinion)
There's no errors/warnings when a policy is being applied to a resource.

File: helm/policies-aws/templates/AWSCAPI.yaml [hunk 1]
```diff
@@ -21,6 +21,26 @@ spec:
...
+        - infrastructure.cluster.x-k8s.io/v1alpha4/AWSClusterControllerIdentity
+        - infrastructure.cluster.x-k8s.io/v1alpha4/AWSClusterRoleIdentity
+        - infrastructure.cluster.x-k8s.io/v1alpha4/AWSClusterStaticIdentity
+        - infrastructure.cluster.x-k8s.io/v1alpha4/AWSMachine
+        - infrastructure.cluster.x-k8s.io/v1alpha4/AWSMachinePool
+        - infrastructure.cluster.x-k8s.io/v1alpha4/AWSMachineTemplate
+        - infrastructure.cluster.x-k8s.io/v1alpha4/AWSManagedMachinePool
+        - controlplane.cluster.x-k8s.io/v1alpha4/AWSManagedControlPlane
+        - bootstrap.cluster.x-k8s.io/v1alpha4/EKSConfigTemplate
+        - infrastructure.cluster.x-k8s.io/v1beta1/AWSCluster
```
[the remark this answers is not in the document]
Comment (AverageMarcus) [reply helm/policies-aws/templates/AWSCAPI.yaml:34]: Confirmed that all the AWS and Azure types we reference have a `v1beta1` version in the latest release. There are a small number of CRDs that don't have a `v1beta1` (such as `AWSManagedCluster`) but we don't use these anyway.

**Review by tfussell — APPROVED** [review]

**Pull request merged**